subsys/bluetooth/controller/ll_sw/ull_scan_aux.c

```diff
@@ -144,6 +146,8 @@ void ull_scan_aux_setup(memq_link_t *link, struct node_rx_hdr *rx)
 				return;
 			}
 
+			is_scan_req = !!ftr->scan_req;
+
```

Comment:
Could this be moved to line 202 instead? Seems to make more sense to update when it needs to be used.